pkg/utils/bytes/bytes.go

```diff
@@ -0,0 +1,17 @@
+package bytes
```

Comment:
Jordan made the suggestion. The idea is to avoid having a large `utils.go` file with everything cramped together.

Reply:
https://go.dev/blog/package-names#bad-package-names
> Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains. This makes it harder for clients to use the package and makes it harder for maintainers to keep the package focused. Over time, they accumulate dependencies that can make compilation significantly and unnecessarily slower, especially in large programs. And since such package names are generic, they are more likely to collide with other packages imported by client code, forcing clients to invent names to distinguish them.

Reply:
IMHO the rule that takes priority here is to not disrupt the local style. This module stands alone, and has much higher standards than core.